evoting/protocol/dkg.go

```diff
@@ -0,0 +1,226 @@
+package protocol
```

Comment:
Is this a copy of the onchain-secrets/protocol/dkg.go?

Reply:
Yes. The three files `dkg.go`, `dkg_struct.go` and `dkg_test.go` are from the onchain-secrets project. Also the `DKGSimulate` function in `lib/utils.go` is taken from `onchain-secrets/protocol/onchain_test.go`.